docs/articles/nunit/getting-started/dotnet-core-and-dotnet-standard.md

```diff
@@ -57,7 +58,7 @@ This is a limitation of Visual Studio, hopefully it will be fixed in a future re
 
 Meanwhile, you can run specific tests using the `--framework` command line option of [dotnet test](https://docs.microsoft.com/en-ca/dotnet/core/tools/dotnet-test?tabs=netcore2x)
 
-#### How do I produce a test results file
+## How do I produce a test results file
```

Comment:
```suggestion
#### How do I produce a test results file
```

Is there a reason why we'd convert this to an H2 instead of keeping it as an H4? 

It seems to be an additional item in the FAQ, and so I'd expect it to have the same size as all the other FAQ responses.

Reply:
This is a section that really doesnt belong here at all.  It should probably be in its own page, or on a page together with How To Do XXXX  , so that is why I lifted it up.